rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -284,6 +284,8 @@ if(BUILD_TESTING AND rosidl_generate_interfaces_ADD_LINTER_TESTS)
     get_filename_component(_cpplint_root "${_output_path}" DIRECTORY)
     ament_cpplint(
       TESTNAME "cpplint_rosidl_generated_py"
+      # the generated code might contain functions with more lines
+      FILTERS "-readability/fn_size"
```

Comment:
ament/ament_lint#87 was necessary to pass the linters since the generated functions are now potentially longer than 
500 lines.